### DISCUSSION TITLE
compatible

### DISCUSSION THREAD

**Discussion by ghost** [P3pp3rF1y/SophisticatedBackpacks#846] [Ideas]
https://github.com/Tiviacz1337/Travelers-Backpack/issues/700

**Comment by P3pp3rF1y** [comment]
Please don't just link to the other repo, the owner over there can completely change the contents and I won't know what was the original message. And others same as me are also forced to open another link if they want to discuss this feature. 
Anyway if your thought is about not having more than one curios back slot that should be doable easily with pack setup and if the thought is about preventing use of both backpacks my thought at least at the moment is that modpack maker can decide to not have multiple backpack mods in the pack if they want to prevent use of multiple backpacks from different mods. 
If you have more to add to this feel free to share, but at this point I feel this is primarily for modpack makers to figure out.
